js/script.js

```diff
@@ -85,10 +126,39 @@ class Store {
 
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static UpdateBook(book) {
+    const books = Store.getBooks();
+
+    books.forEach((b, index) => {
```

Comment:
Add key and value to localStorage so that each time getting all books and searching from those can be optimised.